models/recurly.yml

```diff
@@ -19,7 +19,7 @@ models:
       - name: account_daily_id
         description: Unique account key, based on account id and date of balance data.
         tests: 
-          - unique
+          - unique 
```

Comment:
Should we also add a `not_null` test to these as well?

Reply:
Same comment for the other tests

Reply:
We do have `not_null` tests set up at the staging models, so curious if it'd be redundant?

Reply:
add not-null in